jetty-alpn/jetty-alpn-client/src/main/java/org/eclipse/jetty/alpn/client/ALPNClientConnectionFactory.java

```diff
@@ -68,7 +68,8 @@ public ALPNClientConnectionFactory(Executor executor, ClientConnectionFactory co
             {
                 if (LOG.isDebugEnabled())
                     LOG.debug(x);
-                failure.addSuppressed(x);
+                if (x != failure)
```

Comment:
Isn't this always true in this scenario?
'failure' is from line 57, and isn't passed into / used by anything in the loop.

Reply:
Good catch!